detector/wordpress.go

```diff
@@ -196,27 +216,72 @@ func extractToVulnInfos(pkgName string, cves []WpCveInfo) (vinfos []models.VulnI
 				Link: url,
 			})
 		}
+		for _, id := range vulnerability.References.YouTube {
+			refs = append(refs, models.Reference{
+				Link: fmt.Sprintf("https://www.youtube.com/watch?v=%s", id),
+			})
+		}
+
+		var exploits []models.Exploit
+		for _, id := range vulnerability.References.ExploitDB {
+			exploits = append(exploits, models.Exploit{
+				ExploitType: "wpscan",
+				ID:          id,
```

Comment:
I feel that it's hard to understand what a string of numbers represents until I look at the URL, so do you think it would be better to have a prefix like this?
```suggestion
				ID:          fmt.Sprintf("Exploit-DB: %s", id),
```

Reply:
Thank you for your feedback.
This is the first time someone has looked at my code review in such detail, and I am extremely grateful.